jgrapht-core/src/main/java/org/jgrapht/alg/spanning/PrimMinimumSpanningTree.java

```diff
@@ -103,6 +103,91 @@ public PrimMinimumSpanningTree(Graph<V, E> graph)
 
         return new SpanningTreeImpl<>(minimumSpanningTreeEdgeSet, spanningTreeWeight);
     }
+
+
+    /**
+     * Computes a spanning tree in $O(|V|^2)$ time.
+     *
+     * Note: This method is only recommended for dense graphs.
+     *
+     * @return a spanning tree
+     */
+    @SuppressWarnings("unchecked")
+    public SpanningTree<E> getSpanningTreeDense(){
+        Set<E> minimumSpanningTreeEdgeSet = new HashSet<>(g.vertexSet().size());
+        double spanningTreeWeight = 0d;
+
+        final int N = g.vertexSet().size();
+
+        /*
+         * Normalize the graph
+         *   map each vertex to an integer (using a HashMap)
+         *   keep the reverse mapping  (using an ArrayList)
+         */
+        Map<V, Integer> vertexMap = new HashMap<>();
+        List<V> indexList = new ArrayList<>();
+        for (E e: g.edgeSet()){
```

Comment:
The 'common' practice is:
for(V v : graph.vertexSet()){
  vertexMap.put(v,vertexMap.size());
  indexList.add(v);
}

Reply:
Fixed. Thank you for the observation.